lemur/plugins/lemur_linuxdst/remote_host.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+from lemur.certificates import service
+import os
+
+
+def createCert(name, tempFolder, exportType):
+
+    lemCert = service.get_by_name(name)
+    if not os.path.exists(tempFolder):
+        os.mkdir(tempFolder)
+    if not os.path.exists('{0}/{1}'.format(tempFolder, lemCert.cn)):
+        os.mkdir('{0}/{1}'.format(tempFolder, lemCert.cn))
+    certFile = '{0}/{1}/cert.pem'.format(tempFolder, lemCert.cn)
+    keyFile = '{0}/{1}/priv.key'.format(tempFolder, lemCert.cn)
```

Comment:
@kevgliss  when walking though the link you provide it appears it still wants a file to transfer. Am I overlooking something?